internal/core/application/explorer_utils.go

```diff
@@ -63,3 +65,18 @@ func getBalancesByAsset(unspents []explorer.Utxo) map[string]BalanceInfo {
 	}
 	return balances
 }
+
+func waitForTx(explorerSvc explorer.Service, txid string) error {
+	waitingTime := 1 * time.Second
+	for {
+		_, err := explorerSvc.GetTransaction(txid)
+		if err != nil {
+			if strings.Contains(err.Error(), "Transaction not found") {
```

Comment:
This could be fragile. Better to check the HTTP status code ie. 404? (if is that)

Reply:
ie. if the text of the error changes

Reply:
Wait, the solution is wrong, ie explorer is a general interface while we check against a specific error of the esplora impl.
I'm going to add a new method to the explorer `PollGetKnowTransaction` to the explorer.

Reply:
@tiero done.